spec/lib/tasks/feature_flags_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rake'
+
+RSpec.describe 'Feature Flags', :rake do
```

Comment:
to avoid testing rake tasks, I usually take the approach of moving the ruby code to a ruby class. Something like

```rb
# lib/tasks/feature_flags.rake
require 'lib/tasks/feature_flags'

namespace :feature_flags do
  desc 'Regiser new feature flags from config/feature-flags.yml'
  task initialize: :environment do
    Tasks::FeatureFlags.new.init_all
  end
end
```

```rb
# lib/tasks/feature_flags.rb
module Tasks
  class FeatureFlags
    def init_all
      YAML.load_file('config/feature-flags.yml', fallback: {}).each do |key, _options|
        next if Flipper.exist?(key)

        Flipper.add(key)
      end
    end
  end
end
```

and then you just test the ruby class. BTW if doing this approach you can go back to use constants defined under the corresponding Ruby module/class